index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Remote Work Event</title>
+
+    <!-- font awesome icons -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+
+    <!-- Bootstrap icon -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="./styles/main.css" />
+  </head>
+  <body id="home_page">
+    <!-- ---------- Header Section --------- -->
+    <header>
+      <nav>
+        <div class="top_menu clearfix">
+          <ul class="top_menu_list">
+            <li class="item"><i class="bi bi-facebook"></i></li>
+            <li class="item"><i class="bi bi-twitter"></i></li>
+            <li class="item"><a href="#">Logout</a></li>
+            <li class="item"><a href="#">MyPage</a></li>
+            <li class="item"><a href="#">Korean</a></li>
+          </ul>
+        </div>
+
+        <div class="main_menu">
+          <i class="bi bi-list"></i>
+          <a href="./index.html">
+            <img
+              class="logo"
+              src="./images/logo.png"
+              alt="brand of the company"
+            />
+          </a>
+
+          <ul class="main_menu_list">
+            <li><i class="bi bi-x-lg"></i></li>
+            <li class="item"><a href="./pages/about.html">About</a></li>
+            <li class="item"><a href="#">Program</a></li>
+            <li class="item"><a href="#">Join</a></li>
+            <li class="item"><a href="#">Sponsor</a></li>
+            <li class="item"><a href="#">News</a></li>
+            <li class="">
+              <div class="btn">
+                <a class="cc" href="#">CC Campaign</a>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- ---------- Main Section --------- -->
+    <main class="main">
+      <div class="main__header">
+        <h3>"Hello! Sharing world"</h3>
+        <h1>The future of work</h1>
+      </div>
+
+      <div class="main__middle">
+        <p>
+          Remote is all about how we get the job done today. It looks at issues
+          of work, including the continuing trend for remote working post
+          pandemic, and what that might mean for employers and employees.
+        </p>
+      </div>
+
+      <div class="main__bottom">
+        <h2>2022.7.10(Thus) ~ 20(FRI)</h2>
+        <p>@Great Russell Street, London WC1B 3DG, England</p>
+      </div>
+    </main>
+
+    <!-- ---------- Program Section --------- -->
+    <section class="program container">
+      <h2>Main Program</h2>
+      <div class="line"></div>
+
+      <div class="program_container">
+        <!-- program 1 -->

```

Comment:
- [ ] Main program should have 5 individual cards, you have only 4, kindly add 1 more: 
<img width="813" alt="Screenshot 2022-06-23 at 22 32 01" src="https://user-images.githubusercontent.com/77038610/175369989-029a9198-59c1-4b00-ba30-b1dc0611cbce.png">

Reply:
I added one more card.